addons/web_editor/static/src/js/wysiwyg/widgets/chatgpt_prompt_dialog.js

```diff
@@ -53,6 +53,9 @@ export class ChatGPTPromptDialog extends ChatGPTDialog {
         }
     }
     submitPrompt(ev) {
+        if (!this.state.prompt.trim().length) {
+            return;
+        }
```

Comment:
Can't we _actually_ disable the button? That is use the actual `disabled` html attribute on the button rather than merely using the boostrap class. So adding `t-att-disabled="state.prompt.trim().length || undefined` to the template. Let me know if it works or not.

Reply:
This condition is to prevent submitting prompt in case of pressing `Enter` and not for button. Currently button is already disabled if there is no text and it is not clickable.

Reply:
Shouldn't that be in `onTextareaKeydown` then? So that enter does _not_ call submit if there are no content to submit. Rather than calling submit and then sometimes submit doesn't actually submit. Do you see the difference? I mean conceptually.

Reply:
Yes, conceptually it is correct. submitPrompt shouldn't be called.

Reply:
@dmo-odoo I've moved the condition to `onTextareaKeydown`. Note that i've written this condition inside the above conditions rather than mixing with those two because in that case if text is empty and user presses ctrl + Enter then `ev.stopImmediatePropagation()` will not get called and empty prompt will be sent through hotkey service.